chapter1.md

```diff
@@ -53,6 +53,66 @@ Commands:
   help [cmd]     display help for [cmd]
 ```
 
+The cli provides two commands API and behaviors and each command has its own options.
+
+# Behaviors command
+
+Execute `./bin/cli behaviors -b` to build the behaviors made available, using the config file located at the root of the project. 
+
+The built behaviors, along with a behavior metadata file (`behaviorMetadata.js`), can be found in the `dist` directory which will be created for you if it does not exist in the root of this project.
+
+The following options are available to use with the behaviors command:
+
+```
```

Comment:
as this is intended as a quick getting started guide I think that we don't need to put this output of `help` in here